frontend_tests/puppeteer_tests/drafts.ts

```diff
@@ -8,8 +8,16 @@ async function wait_for_drafts_to_disappear(page: Page): Promise<void> {
     await page.waitForSelector("#draft_overlay.show", {hidden: true});
 }
 
-async function wait_for_drafts_to_appear(page: Page): Promise<void> {
+async function wait_for_drafts_to_appear(
+    page: Page,
+    has_any_draft: boolean = false,
+): Promise<void> {
     await page.waitForSelector("#draft_overlay.show");
+    if (has_any_draft) {
+        await page.waitForFunction(() =>
+            document?.activeElement?.classList.contains("draft-info-box"),
```

Comment:
I don’t think the focus change is linked to the animation in any way.

Reply:
And indeed, by exaggerating the animation you can see that this still fails.

```diff
--- a/static/styles/app_components.css
+++ b/static/styles/app_components.css
@@ -352,7 +352,7 @@ div.overlay {
     overflow: hidden;
 
     .overlay-content {
-        transform: translateY(20px);
+        transform: translateY(500px);
         transition: transform 0.2s ease-in;
         z-index: 102;
     }
```

Reply:
Oh, Okay. 
So it was not failing only because of the delay from both `waitForFunction` and `waitForSelector`.